code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -39,6 +39,14 @@
 		r_store,
 		w_uniform)
 
+/mob/living/carbon/human/get_nonclothing_items() //pockets, hands, belt

```

Comment:
This is the inverse of get_clothing_items() which gets everything except those three types